apps/comps/components/clipboard/index.tsx

```diff
@@ -9,10 +13,31 @@ export const Clipboard = ({
   text: string;
   className?: string;
 }) => {
+  const [copied, setCopied] = React.useState(false);
+  const [, copyToClipboard] = useCopyToClipboard();
+
+  const handleCopy = async () => {
+    copyToClipboard(text);
+    setCopied(true);
+    toast.success("Wallet Address copied to clipboard");
```

Comment:
it doesnt look like this is being applied? i dont see a toast when i copy the agent's allet